index.html

```diff
@@ -12,6 +12,7 @@
   </head>
   <body>
     <div id="app"></div>
+    <script src="https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/jsoneditor.js"></script>
```

Comment:
Could we move this import to the `config.js`? This way all the custom functionality would be self-contained there